dm/loader/lightning.go

```diff
@@ -412,5 +412,16 @@ func (l *LightningLoader) status() *pb.LoadStatus {
 
 // Status returns the unit's current status.
 func (l *LightningLoader) Status(_ *binlog.SourceStatus) interface{} {
+	go l.printStatus()
 	return l.status()
```

Comment:
Sorry that we don't need to insist on using another goroutine. s := l.status(), using s to log and returning it is easier.